source/upgrade/important-upgrade-notes.rst

```diff
@@ -19,15 +19,40 @@ Important Upgrade Notes
-|                                                    | **IMPORTANT:** Session length configuration settings have changed from using a unit of *days* to *hours*. Instances using a config.json file or a database       |
+| v7.1                                               | A new configuration option ``MaxImageDecoderConcurrency`` indicates how many images can be decoded concurrently at once. The default is -1, and the value        |
+|                                                    | indicates the number of CPUs present. This affects the total memory consumption of the server. The maximum memory of a single image is dictated by               |
+|                                                    | ``MaxImageResolution * 24 bytes``. Therefore, we recommend that ``MaxImageResolution * MaxImageDecoderConcurrency * 24`` should be less than the allocated       |
+|                                                    | memory for image decoding.                                                                                                                                       |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Mattermost v7.1 introduces schema changes in the form of a new column and its index. The following notes our test results for the schema changes:                |                                                                                                                                                                                                             
+|                                                    |                                                                                                                                                                  |
+|                                                    | - MySQL 12M Posts, 2.5M Reactions - ~1min 34s (instance: PC with 8 cores, 16GB RAM)                                                                              |
+|                                                    | - PostgreSQL 12M Posts, 2.5M Reactions - ~1min 18s (instance: db.r5.2xlarge)                                                                                     |
+|                                                    |                                                                                                                                                                  |
+|                                                    | Customers wanting to run the SQL queries before-hand can run these queries. This is fully backwards-compatible, but will obtain a lock on ``Reactions`` table,   |
```

Comment:
Posting the text update here as a comment rather than as a suggestion because I suspect I'll break the table formatting otherwise.

"This is fully backwards-compatible and obtains a lock on ``Reactions`` table, so users' reactions posted during this time won't be reflected in the database until the migrations are complete."